Locales/Chinese_zh.json

```diff
@@ -0,0 +1,890 @@
+  "Time passed!": "时间已过!",
+  "Round lost!": "游戏失败!",
+  "Kill'em All!": "杀光他们!",
+  "Kill the Necromancer": "杀死死灵法师!",
+  "Parkour": "跑酷",
+  "Save the Princess!": "拯救公主!",
+  "Save the Princess": "拯救公主",
+  "Take the Hall": "占领大厅",
+  "All Gamemodes": "所有游戏模式",
+  "Challenges": "挑战",
+  "Capture the Flag": "夺旗模式",
+  "CTF": "夺旗模式",
+  "Team Deathmatch": "团队死斗",
+  "Take the Halls": "占领大厅",
+  "TTH": "占领大厅",
+  "Challenge": "挑战",
+  "Build whatever your heart desires. Respect other folk's creations!": "随心所欲地建造吧. 但也请尊重其他玩家的创作!",
+  "IP:Port": "IP:端口号",
+  "Password:": "密码:",
+  "Heads Pack Available On Steam!": "头像包现已登陆Steam!",
+  "Get exclusive cosmetic heads to increase your customisation options, and stand out on the battlefield!\n\nCurrently only available through steam as a DLC. If there's enough interest we can investigate non-steam options.\n\nProceeds go to continued development of King Arthur's Gold.": "获得独家装饰性的头像来增加你的自定选项, 并在战场上更加出众!\n\n目前仅在Steam上作为DLC可购买. 如果有足够多的人产生兴趣的话我们将会研究非Steam的购买方式.\n\n收入将用于继续开发亚瑟王的金子游戏.",
+  "Find Out More!": "找到更多!",
+  "Close": "关闭",
+  "Username:": "用户名:",
+  "Remember me": "记住我",
+  "Create Account": "新建账户",
+  "Offline": "离线模式",
+  "Forgot?": "忘记了?",
+  "PLAY": "开始游戏",
+  "Solo": "单人模式",
+  "Multiplayer": "多人模式",
+  "Settings": "设置",
+  "How to video": "教程视频",
+  "Forum": "论坛",
+  "News": "新闻",
+  "Discord": "Discord",
+  "IRC": "IRC",
+  "Wiki": "维基",
+  "Set account e-mail": "设置账户电子邮件",
+  "Quit": "退出",
+  "Your KAG account e-mail and password": "你的亚瑟王的金子的账户电子邮箱和密码",
+  "You can use these credentials to login on kag2d.com and kagforums.com": "你可以用这些信息来登录kag2d.com 和 kagforums.com",
+  "E-mail:": "电子邮箱",
+  "Submit": "提交",
+  "Simple Join": "快速加入",
+  "Browse servers": "查看服务器列表",
+  "Connect to...": "连接到...",
+  "New Game From THD:": "来自THD的新游戏:",
+  "Pick Game Size:": "选择游戏大小:",
+  "<8 Players": "少于8人",
+  "8-20 Players": "8至20人",
+  ">20 Players": "多于20 人",
+  "Don't Care": "随便",
+  "Choose Game Mode:": "选择游戏模式:",
+  "Sandbox": "沙盒",
+  "Co-op Challenge": "合作挑战",
+  "Custom Content": "自定义内容",
+  "Play!": "开始游戏!",
+  "Controls": "控制",
+  "Video": "视频",
+  "Sound": "声音",
+  "Misc": "杂项",
+  "Clantag:": "战队名:",
+  "Gender:": "性别:",
+  "Male": "男性",
+  "Female": "女性",
+  "BUY {HEADS_PACK}": "购买 {HEADS_PACK}",
+  "Standard": "标准",
+  "Flags of The World": "世界旗",
+  "Head packs:": "头像包:",
+  "Head:": "头部:",
+  "Change keys:": "更改按键:",
+  "Smooth shader": "平滑着色",
+  "Faster graphics": "快速图像",
+  "Uncap framerate": "无上限FPS",
+  "VSync - Requires Restart": "垂直同步 - 需要重启生效",
+  "Game music": "游戏音乐",
+  "Sound Volume:": "音效音量:",
+  "Music Volume:": "音乐音量:",
+  "Volume:": "音量:",
+  "Menu music": "菜单音乐",
+  "Swap audio channels": "交换左右声道",
+  "Language (EXPERIMENTAL):": "语言 (实验支持):",
+  "Fixed camera": "固定视角",
+  "Kids safe": "适合儿童",
+  "Chat Filter": "聊天消息过滤",
+  "Show Team-Mates Names": "显示队伍成员的名字",
+  "Show Minimap": "显示小地图",
+  "Show Chat Bubbles": "显示聊天气泡",
+  "Snap Camera To Pixel": "拉近视角至像素",
```

Comment:
this means that the camera always has its borders on the edge of pixels, but I'm not sure how to translate that